Library/Formula/qpm.rb

```diff
@@ -0,0 +1,18 @@
+class Qpm < Formula
+  desc "Package manager for Qt applications"
+  homepage "https://www.qpm.io"
+  url "https://github.com/Cutehacks/qpm.git", :tag => "v0.10.0",
```

Comment:
Would you mind putting the `tag` on its own line? Thanks. Trying to keep URL/tag/revision lines more or less in sync so people can just glance down a formula rather than quite far across.